tests/e2e/keys.go

```diff
@@ -35,8 +36,11 @@ func createMemoryKey() (mnemonic string, info *keyring.Info, err error) {
 	return mnemonic, account, nil
 }
 
-func createMemoryKeyFromMnemonic(mnemonic string) (*keyring.Info, error) {
-	kb, err := keyring.New("testnet", keyring.BackendMemory, "", nil)
+func createMemoryKeyFromMnemonic(mnemonic string) (*keyring.Record, error) {
+
+	cdc := params.MakeTestEncodingConfig().Codec
```

Comment:
There should already be a private global for the codec -- let's use that instead of creating a new variable here.

Reply:
ah that's great! how do i access it?